experimental/sdk/webpubsub-graphql-subscribe/src/demos/demo.ts

```diff
@@ -0,0 +1,77 @@
+// Modified From https://github.com/apollographql/docs-examples/blob/7105d77acfc67d6cb4097cc27a7956051ec0c1b5/server-subscriptions-as3/index.js
+import {create_webpubsub_subscribe_server} from '../WpsWebSocketServer'
+import { WpsPubSub } from '../azure-wps-pubsub';
+import { ApolloServer, gql } from "apollo-server-express";
+import { makeExecutableSchema } from "@graphql-tools/schema";
+import { SubscriptionServer } from 'subscriptions-transport-ws';
+import { PubSub } from "graphql-subscriptions";
+import { createServer } from "http";
+import { execute, subscribe } from "graphql";
+import express from 'express';
+import {config} from "../utils"
+
+const webpubsub_conn_string = "<web-pubsub-connection-string>";
+
+// when useWPS = false, subscription server will be launched without Azure WebPub support
+const useWPS = true;
+
+const pubsub = useWPS ? new WpsPubSub(webpubsub_conn_string) : new PubSub();
+
+let currentNumber = 0;
+
+// Schema definition
+const typeDefs = gql`
+	type Query { currentNumber: Int }
+	type Subscription { numberIncremented: Int }
+`;
+
+// Resolver map
+const resolvers = {
+	Query: {
+		currentNumber() { return currentNumber; }
+	},
+	Subscription: {
+		numberIncremented: {
+			subscribe: () => pubsub.asyncIterator(['NUMBER_INCREMENTED']),    // subscribe: asyncIterator([<event-name-1>, ...])
+		},
+	}
+};
+
+const schema = makeExecutableSchema({ typeDefs, resolvers });
+
+function incrementNumber() {
+	currentNumber++;
+	pubsub.publish('NUMBER_INCREMENTED', { numberIncremented: currentNumber });     // publish: publish(<event-name>, <event-data>)
+	setTimeout(incrementNumber, 1000);
+}
+
+async function main() {
+	// Create a HTTP server
+	const app = express();
+	const httpServer = createServer(app);	// http.createServer
+	
+	// Create and instlal Apollo Server
+	const apolloServer = new ApolloServer({schema});
+	await apolloServer.start();
+	app.use(apolloServer.getMiddleware());
+
+	// Create a Subscription Server
+	if (useWPS) {
+		await create_webpubsub_subscribe_server(apolloServer, schema, pubsub as WpsPubSub, webpubsub_conn_string);
+	} else {
+		SubscriptionServer.create(
```

Comment:
why not using `apolloServer.installSubscriptionHandlers(wpsServer);`?

Reply:
`ApolloServer` and `SubscriptionServer` is two separate component.
Actually `SubscriptionServer`  can work independently.
`installSubscriptionHandlers` couples `SubscriptionServer` and `ApolloServer` into a `ApolloServer`.
To decouple these two components and make logic more clear, I didn't use `installSubscriptionHandlers`.

Reply:
I see, how about having a method to return the wps options instead, e.g.
```js
var createOptions = useWPS ? await getWebPubSubServerOptions(...) : { server: httpServer, path: apolloServer.graphqlPath };
SubscriptionServer.create(
{ schema, execute, subscribe },
createOptions );


```

Reply:
fixed